src/instructlab/model/train.py

```diff
@@ -130,7 +133,7 @@ def clickpath_setup(is_dir: bool) -> click.Path:
 )
 @click.option(
     "--device",
-    type=click.Choice(["cpu", "cuda", "hpu"]),
+    type=click.Choice(["cpu", "cuda", "hpu", "mps"]),
```

Comment:
from a UX POV, how do we communicate to Mac users they should use MPS?

Reply:
MPS is Metal Performance Shaders developed by Apple. I can add something to the help text and maybe the readme?